easybuild/easyconfigs/k/kalign2/kalign2-2.0.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
+version = '2.0.4'
+
+homepage = ''
```

Comment:
@sassy-crick isn't this effectively the homepage?: https://msa.sbc.su.se/cgi-bin/msa.cgi

Reply:
Correct. I forgot to put that in, apologies for that. This should be the one for `kalign2-version2`